### PR TITLE
Configure 24 resources without testing

### PR DESCRIPTION
**Pull request by sergenyalcin**
### Description of your changes

Configure 24 resources without testing. Related issues:

https://github.com/upbound/provider-aws/issues/50
https://github.com/upbound/provider-aws/issues/40
https://github.com/upbound/provider-aws/issues/64

I have:

- [x] Run `make reviewable test` to ensure this PR is ready for review.

### CODE REVIEW

File: config/externalnamenottested.go [hunk 1]
```diff
@@ -335,4 +335,67 @@ var ExternalNameNotTestedConfigs = map[string]config.ExternalName{
...
+	"aws_s3control_bucket_lifecycle_configuration": config.IdentifierFromProvider,
+	// S3 Control Bucket Policies can be imported using the Amazon Resource Name (ARN)
+	"aws_s3control_bucket_policy": config.IdentifierFromProvider,
+
+	// elasticbeanstalk
+	//
+	// Elastic Beanstalk Applications can be imported using the name
+	"aws_elastic_beanstalk_application": config.NameAsIdentifier,
+	// No import
+	"aws_elastic_beanstalk_application_version": config.IdentifierFromProvider,
```
Comment (ulucinar) [inline config/externalnamenottested.go:387]: Although import docs are missing for the resource, I suspect this resource is using the `name` argument as a naming field. We may also consider configuring accordingly and fix in the tests. What do you think? Even if don't know the exact syntax for the ID, at least we can consider adding two TODOs for: 1. We check the resource ID when we are testing the resource and try to perform the normalization during testing 2. Stating that the API is currently not normalized.

File: config/externalnamenottested.go [hunk 2]
```diff
@@ -335,4 +335,67 @@ var ExternalNameNotTestedConfigs = map[string]config.ExternalName{
...
+	// No import
+	"aws_elastic_beanstalk_configuration_template": config.IdentifierFromProvider,
```
Comment (ulucinar) [inline config/externalnamenottested.go:389]: Same as the above comment regarding this resource's `name` argument.

File: config/externalnamenottested.go [hunk 3]
```diff
@@ -335,4 +335,67 @@ var ExternalNameNotTestedConfigs = map[string]config.ExternalName{
...
+	//
+	// Route53 Recovery Readiness recovery groups can be imported via the recovery group name
+	"aws_route53recoveryreadiness_recovery_group": config.ParameterAsIdentifier("recovery_group_name"),
+	// Route53 Recovery Readiness resource set name can be imported via the resource set name
+	"aws_route53recoveryreadiness_resource_set": config.ParameterAsIdentifier("resource_set_name"),
+
+	// s3control
+	//
+	//
+	"aws_s3_access_point": config.IdentifierFromProvider,
```
[the remark this answers is not in the document]
Comment (sergenyalcin) [reply config/externalnamenottested.go:379]: TODO added!

**Review by ulucinar — APPROVED** [review]
Thank you @sergenyalcin.

**Pull request merged**